meveo-admin/ejbs/src/main/java/org/meveo/service/crm/impl/JSONSchemaIntoJavaClassParser.java

```diff
@@ -511,10 +511,8 @@ private FieldDeclaration getMatrixField(CustomFieldTemplate cft, CompilationUnit
 		case MULTI_VALUE:
 		case SECRET:
 		case STRING:
-		case TEXT_AREA:
-			vd.setType("MeveoMatrix<String>");
```

Comment:
This line should not have been deleted, it's the fall through case of the other types

Reply:
I am not understand  this , why i can not delete this one 
 if it is not required. 
what i have to do instead of delete this value.

Reply:
If you delete the vd.setType(...) statement, then all the other cases will fall to the default case, which is noop.

Here you only need to delete the case statement but not the instruction